rebasehelper/plugins/checkers/rpminspect.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you rebase your package to the latest version
+# Copyright (C) 2013-2019 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hráček <phracek@redhat.com>
+#          Tomáš Hozza <thozza@redhat.com>
+#          Nikola Forró <nforro@redhat.com>
+#          František Nečas <fifinecas@seznam.cz>
+
+import os
+import json
+
+from typing import Any, Dict, Tuple
+
+from rebasehelper.exceptions import CheckerNotFoundError, RebaseHelperError
+from rebasehelper.helpers.rpm_helper import RpmHelper
+from rebasehelper.helpers.process_helper import ProcessHelper
+from rebasehelper.plugins.checkers import BaseChecker
+
+
+class Rpminspect(BaseChecker):  # pylint: disable=abstract-method
+    """An abstract class representing rpminspect checker (either RPM or SRPM).
+
+    For both types of checkers the data stored in results_store
+    looks as follows:
+
+    {
+        'checks': {
+            'package1': {
+                'BAD': 5,
+                'VERIFY': 3,
+                'OK': 1,
+                'INFO': 1
+            },
+            'package2': {
+                'BAD': 1,
+                'VERIFY': 3,
+                'OK': 5,
+                'INFO': 1
+            }
+        },
+        'path': rebase-helper-results/checkers/rpminspect,
+        'files': [
+            'package1.json',
+            'package2.json'
+        ]
+    }
+    """
+    DEFAULT: bool = True
+
+    CMD: str = 'rpminspect'
+
+    BAD_CHECK: str = 'BAD'
+    VERIFY_CHECK: str = 'VERIFY'
+    OK_CHECK: str = 'OK'
+    INFO_CHECK: str = 'INFO'
```

Comment:
I think it would be better to group these into a list, so you can iterate over all of them later.